tests/schemes/test_thrift.py

```diff
@@ -62,98 +54,56 @@ def server(io_loop):  # need io_loop fixture for listen() to work
     return server
 
 
-@pytest.fixture(params=[False, True], ids=['server_thrift', 'server_thriftrw'])
-def use_thriftrw_server(request):
-    return request.param
-
-
-@pytest.fixture(params=[False, True], ids=['client_thrift', 'client_thriftrw'])
-def use_thriftrw_client(request):
-    return request.param
-
-
 @pytest.fixture
-def ThriftTest(use_thriftrw_server):
+def ThriftTest():
     """Used by servers to register endpoints for ThriftTest."""
-
-    if use_thriftrw_server:
-        return thrift.load(
-            'tests/data/idls/ThriftTest.thrift',
-        ).ThriftTest
-    else:
-        return _ThriftTest
+    return thrift.load(
+        'tests/data/idls/ThriftTest.thrift',
+    ).ThriftTest
 
 
 @pytest.fixture
-def SecondService(use_thriftrw_server):
+def SecondService():
     """Used by servers to register endpoints for SecondService."""
-
-    if use_thriftrw_server:
-        return thrift.load(
-            'tests/data/idls/ThriftTest.thrift',
-        ).SecondService
-    else:
-        return _SecondService
+    return thrift.load(
+        'tests/data/idls/ThriftTest.thrift',
+    ).SecondService
 
 
 @pytest.fixture
-def server_ttypes(use_thriftrw_server):
+def server_ttypes():
     """Provides access to generated types for the server."""
-
-    if use_thriftrw_server:
-        return thrift.load(
-            path='tests/data/idls/ThriftTest.thrift',
-        )
-    else:
-        return _ttypes
+    return thrift.load(
+        path='tests/data/idls/ThriftTest.thrift',
+    )
 
 
 @pytest.fixture
-def client_ttypes(use_thriftrw_client):
+def client_ttypes():
     """Provides access to generated types for the server."""
-
-    if use_thriftrw_client:
-        return thrift.load(
-            path='tests/data/idls/ThriftTest.thrift',
-        )
-    else:
-        return _ttypes
+    return thrift.load(
+        path='tests/data/idls/ThriftTest.thrift',
+    )
 
 
 @pytest.fixture
-def service(server, use_thriftrw_client):
+def service(server):
     """Used by clients to build requests to ThriftTest."""
-
-    if use_thriftrw_client:
-        return thrift.load(
-            path='tests/data/idls/ThriftTest.thrift',
-            service='server',
-            hostport=server.hostport,
-        ).ThriftTest
-    else:
-        return thrift_request_builder(
-            service='server',
-            thrift_module=_ThriftTest,
-            hostport=server.hostport,
-        )
+    return thrift.load(
+        path='tests/data/idls/ThriftTest.thrift',
+        service='server',
+        hostport=server.hostport,
```

Comment:
since we already support hostport on the tchannel.thrift call method. There is no need to have two fixture of this IDL.